template/src/api/types/comic.types.ts

```diff
@@ -0,0 +1,20 @@
+export interface ComicBE {
```

Comment:
Maybe it's a good idea to use `*.d.ts` extension instead of `*.types.ts` for files consisting of types declarations only (no implementation codes).

If you agree then it can be a separate Issue/PR.

Reply:
I think `d.ts` files were meant to provide types to incorrectly typed / untyped JS libraries.
[This is](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) what official examples looks like.
Buta at the same time I'm not 100% sure about that, or if there is another con of using it differently.
leaving for now